packages/tocco-ui/src/Panel/PanelGroup.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import StyledPanelGroup from './StyledPanelGroup'
+
+class PanelGroup extends React.Component {
+  state = {
+    openPanelId: this.props.openPanelId
```

Comment:
I'd rename `openPanelId` to `openPanelIndex` (both the state property and the prop; makes it easier to understand quickly what it represents -> `...Index` indicates that it's an integer starting at zero whereas `...Id` could be anything)